homeassistant/components/starlink/coordinator.py

```diff
@@ -54,13 +58,16 @@ def __init__(self, hass: HomeAssistant, name: str, url: str) -> None:
     async def _async_update_data(self) -> StarlinkData:
         async with asyncio.timeout(4):
             try:
-                status = await self.hass.async_add_executor_job(
-                    status_data, self.channel_context
-                )
-                location = await self.hass.async_add_executor_job(
-                    location_data, self.channel_context
+                status, location, sleep = await asyncio.gather(
+                    self.hass.async_add_executor_job(status_data, self.channel_context),
+                    self.hass.async_add_executor_job(
+                        location_data, self.channel_context
+                    ),
+                    self.hass.async_add_executor_job(
```

Comment:
Don't create concurrent executor jobs. We're not allowed to create more than one executor job at a time. Please run all sync work inside a single function that we schedule once on the executor.